autogpt_platform/frontend/src/components/monitor/FlowRunInfo.tsx

```diff
@@ -1,79 +1,158 @@
+          <div>
+            <CardTitle>
+              {flow.name} <span className="font-light">v{flow.version}</span>
+            </CardTitle>
+            <p className="mt-2">
+              Agent ID: <code>{flow.id}</code>
+            </p>
+            <p className="mt-1">
+              Run ID: <code>{flowRun.id}</code>
+            </p>
+          </div>
+          <div className="flex space-x-2">
+            {flowRun.status === "running" && (
+              <Button onClick={handleStopRun} variant="destructive">
+                <IconSquare className="mr-2" /> Stop Run
+              </Button>
+            )}
+            <Button onClick={() => setIsOutputOpen(true)} variant="outline">
+              <ExitIcon className="mr-2" /> View Outputs
+            </Button>
+            <Link
+              className={buttonVariants({ variant: "outline" })}
+              href={`/build?flowID=${flow.id}`}
+            >
+              <Pencil2Icon className="mr-2" /> Edit Agent
+            </Link>
+          </div>
+        </CardHeader>
+        <CardContent>
+          <div>
+            <strong>Status:</strong>{" "}
+            <FlowRunStatusBadge status={flowRun.status} />
+          </div>
+          <p>
+            <strong>Started:</strong>{" "}
+            {moment(flowRun.startTime).format("YYYY-MM-DD HH:mm:ss")}
           </p>
-          <p className="mt-1">
-            Run ID: <code>{flowRun.id}</code>
+          <p>
+            <strong>Finished:</strong>{" "}
+            {moment(flowRun.endTime).format("YYYY-MM-DD HH:mm:ss")}
           </p>
-        </div>
-        <div className="flex space-x-2">
-          {flowRun.status === "running" && (
-            <Button onClick={handleStopRun} variant="destructive">
-              <IconSquare className="mr-2" /> Stop Run
-            </Button>
-          )}
-          <Link
-            className={buttonVariants({ variant: "outline" })}
-            href={`/build?flowID=${flow.id}`}
-          >
-            <Pencil2Icon className="mr-2" /> Edit Agent
-          </Link>
-        </div>
-      </CardHeader>
-      <CardContent>
-        <div>
-          <strong>Status:</strong>{" "}
-          <FlowRunStatusBadge status={flowRun.status} />
-        </div>
-        <p>
-          <strong>Started:</strong>{" "}
-          {moment(flowRun.startTime).format("YYYY-MM-DD HH:mm:ss")}
-        </p>
-        <p>
-          <strong>Finished:</strong>{" "}
-          {moment(flowRun.endTime).format("YYYY-MM-DD HH:mm:ss")}
-        </p>
-        <p>
-          <strong>Duration (run time):</strong> {flowRun.duration} (
-          {flowRun.totalRunTime}) seconds
-        </p>
-        {/* <p><strong>Total cost:</strong> €1,23</p> */}
-      </CardContent>
-    </Card>
+          <p>
+            <strong>Duration (run time):</strong> {flowRun.duration}s
+          </p>
+          {/* <p><strong>Total cost:</strong> €1,23</p> */}
```

Comment:
nit: I know this might not be your change, but can you remove this comment if you end up adding more commits on this PR ? :D